shared/actions/devices.js

```diff
@@ -16,11 +17,21 @@ export function loadDevices () : AsyncAction {
       param: {},
       incomingCallMap: {},
       callback: (error, devices) => {
-        dispatch({
-          type: Constants.showDevices,
-          payload: error || devices,
-          error: !!error
-        })
+        // Flow is weird here, we have to give it true or false directly instead of just giving it !!error
```

Comment:
Blurg. Prefer the conciseness of the previous. There's no flow fix for this? Can we make a ticket on their side?